pennylane/qnode.py

```diff
@@ -426,9 +426,7 @@ def _validate_adjoint_method(device):
         # need to inspect the circuit measurements to ensure only expectation values are taken. This
         # cannot be done here since we don't yet know the composition of the circuit.
 
-        required_attrs = ["_apply_operation", "_apply_unitary", "adjoint_jacobian"]
-        supported_device = all(hasattr(device, attr) for attr in required_attrs)
-        supported_device = supported_device and device.capabilities().get("returns_state")
+        supported_device = device.capabilities().get("provides_adjoint_method")

```

Comment:
Good point Antal. Yes, that would be a required update to Lightning too. Right now the checks use the existence of private methods to test for support, but I think being explicit makes sense for this. Likely this will also go away with the new device API changes following the ADR from last q.